hkmc2DiffTests/src/test/scala/hkmc2/MLsDiffMaker.scala

```diff
@@ -294,6 +294,7 @@ abstract class MLsDiffMaker extends DiffMaker:
       val flo = floan.typeProd(trm)
       floan.solveConstraints()
       floan.expandTerms()
+      floan.expandLeadingDotSels()
```

Comment:
This shoudl be rolled into the general `expandTerms` function. There's no sense in separating them at this level.